### PR TITLE
Add Gumroad

### PR DESCRIPTION
**Pull request by hkamran80**

### CODE REVIEW

File: img/g/gumroad.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="none"><circle cx="16" cy="16" r="16" fill="#FF90E8"/><path fill="#000" d="M14.61 24.064c-4.686 0-7.443-3.76-7.443-8.436 0-4.86 3.033-8.802 8.822-8.802 5.974 0 7.995 4.034 8.087 6.327h-4.319c-.092-1.284-1.195-3.21-3.86-3.21-2.848 0-4.687 2.476-4.687 5.502s1.838 5.501 4.687 5.501c2.574 0 3.676-2.017 4.136-4.034h-4.136v-1.65h8.678v8.435h-3.807v-5.318c-.276 1.925-1.47 5.684-6.157 5.684Z"/></svg>
```
Comment (Carlgo11) [inline img/g/gumroad.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><circle cx="16" cy="16" r="16" fill="#FF90E8"/><path d="M14.61 24.06c-4.69 0-7.44-3.76-7.44-8.43 0-4.86 3.03-8.8 8.82-8.8 5.97 0 8 4.03 8.09 6.32h-4.32c-.1-1.28-1.2-3.2-3.86-3.2-2.85 0-4.69 2.47-4.69 5.5s1.84 5.5 4.69 5.5c2.57 0 3.67-2.02 4.13-4.04H15.9v-1.65h8.68v8.44h-3.81v-5.32c-.28 1.92-1.47 5.68-6.16 5.68Z"/></svg>
```

File: entries/g/gumroad.com.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "Gumroad": {
+      "domain": "gumroad.com",
+      "tfa": [
+        "email"
+      ],
+      "documentation": "https://help.gumroad.com/article/292-account-login-security",
```
Comment (Carlgo11) [inline entries/g/gumroad.com.json]:
```suggestion
      "documentation": "https://help.gumroad.com/article/292--",
```

File: entries/g/gumroad.com.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      "notes": "2FA is on by default for all users",
```
Comment (Carlgo11) [inline entries/g/gumroad.com.json]: This information is not of much use for users as the documentation page already explains this.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by hkamran80** [comment]
Out of curiosity, what tool do you use to simplify the SVGs?

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
@hkamran80 I first copy the code to [SVGO](https://jakearchibald.github.io/svgomg/) where I enable everything except `Remove xmlns` and fiddle with the number precision; Then I download the new XML code and make some final adjustments in a text editor and sometimes I also use Inkscape to fix the viewport.

**Comment by hkamran80** [comment]
> @hkamran80 I first copy the code to [SVGO](https://jakearchibald.github.io/svgomg/) where I enable everything except `Remove xmlns` and fiddle with the number precision; Then I download the new XML code and make some final adjustments in Notepad and sometimes Inkscape.

Thanks!

**Pull request merged**